crates/sui-indexer-alt-jsonrpc/src/args.rs

```diff
@@ -7,7 +7,9 @@ use sui_indexer_alt_metrics::MetricsArgs;
 use sui_pg_db::DbArgs;
 use url::Url;
 
-use crate::{data::system_package_task::SystemPackageTaskArgs, NodeArgs, RpcArgs};
+use crate::{NodeArgs, RpcArgs};
+
+pub use sui_indexer_alt_reader::system_package_task::SystemPackageTaskArgs;
```

Comment:
I read that by `pub use` here SystemPackageTaskArgs is re-exported for references within this crate, wondering any reason of re-exporting here instead of in `lib.rs`
- SystemPackageTaskArgs is imported in lib.rs
- shorter `use` on the reference end

Reply:
It's re-exported here because this is the module that includes all the other arguments.